source/_integrations/bosch_alarm.markdown

```
@@ -47,16 +47,32 @@ The following {% term entities %} are provided:
-### Sensor
+## Actions
 
-A sensor is provided per area that lists how many points are currently in a faulted state.
+The integration provides the following actions.
+
+### Action: Set panel date and time
+
+The `bosch_alarm.set_date_time` action is used to update the date and time on the panel.
+
+- **Data attribute**: `config_entry_id`
+   - **Description**: The ID of the config entry to get the schedule from.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)